pkg/kubectl/cmd/delete.go

```diff
@@ -150,7 +150,7 @@ func NewCmdDelete(f cmdutil.Factory, out, errOut io.Writer) *cobra.Command {
 	usage := "containing the resource to delete."
 	cmdutil.AddFilenameOptionFlags(cmd, &options.FilenameOptions, usage)
 	cmd.Flags().StringVarP(&options.Selector, "selector", "l", "", "Selector (label query) to filter on.")
-	cmd.Flags().BoolVar(&options.DeleteAll, "all", false, "Select all resources in the namespace of the specified resource types.")
+	cmd.Flags().BoolVar(&options.DeleteAll, "all", false, "Delete all resources in the namespace of the specified resource types. It will delete the uninitialized objects as well, unless --include-uninitialized=false is set explicitly.")
```

Comment:
@dixudx how about this:
* in the description of `--all`, we add "including uninitialized objects". For example, this line will become "Delete all resources, including uninitialized ones, in the namespace of the specified resource types."

* in the description of `--include-uninitialized`, we say "the value of this flag overrides the flags that make the kubectl command apply to uninitialized objects, e.g., --all"

Reply:
SGTM.

Reply:
@caesarxuchao @smarterclayton Updated the flag description. PTAL. Thanks.